src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -390,7 +390,7 @@ public static String getPathWithoutBucket(String path) {
      * To transform other types of string URLs into Paths, use IOUtils.getPath instead.
      */
     public static java.nio.file.Path getPathOnGcs(String gcsUrl) {
-        final String[] split = gcsUrl.split("/");
+        final String[] split = gcsUrl.split("/", -1);
```

Comment:
Add comment explaining why we pass in -1, just in case posterity is puzzled by this.